Assets/Scripts/SubView/ILayerChehkerView.cs

```diff
@@ -0,0 +1,6 @@
+﻿using UnityEngine;
+
+public interface ILayerChehkerView
+{
+    public bool IsInLayerMask(LayerMask mask, GameObject obj);    
```

Comment:
Redundant public modifier

Reply:
I didn't notice it before, thanks, I'll delete it